docs/src/main/paradox/user/lang/kotlin/gettingstarted.md

```diff
@@ -0,0 +1,288 @@
+                    <execution>
+                        <id>default-testCompile</id>
+                        <phase>none</phase>
+                    </execution>
+                    <execution>
+                        <id>java-compile</id>
+                        <phase>compile</phase>
+                        <goals> <goal>compile</goal> </goals>
+                    </execution>
+                    <execution>
+                        <id>java-test-compile</id>
+                        <phase>test-compile</phase>
+                        <goals> <goal>testCompile</goal> </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.xolstice.maven.plugins</groupId>
+                <artifactId>protobuf-maven-plugin</artifactId>
+                <version>0.6.1</version>
+                <configuration>
+                    <protocExecutable>/usr/local/bin/protoc</protocExecutable>
+                    <protocArtifact>com.google.protobuf:protoc:3.9.1:exe:${os.detected.classifier}</protocArtifact>
+                </configuration>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>com.google.cloud.tools</groupId>
+                <artifactId>jib-maven-plugin</artifactId>
+                <version>1.7.0</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>build</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <to>
+                        <image>${repo.name}</image>
+                        <credHelper></credHelper>
+                        <tags>
+                            <tag>${tag.version}</tag>
+                        </tags>
+                    </to>
+                    <container>
+                        <mainClass>${main.class}</mainClass>
+                        <jvmFlags>
+                            <jvmFlag>-XX:+UseG1GC</jvmFlag>
+                            <jvmFlag>-XX:+UseStringDeduplication</jvmFlag>
+                        </jvmFlags>
+                        <ports>
+                            <port>8080</port>
+                        </ports>
+                    </container>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    
+</project>
+```
+@@@
+
+Subsequent source locations and build commands will assume the above Maven project, and may need to be adapted to your particular build tool and setup.
+
+## Cloudstate CLI example
```

Comment:
As it's mentiond here, the Cloudstate CLI could be in general be described in the documentation, and not only in the kotlin language support? (even if outside of the Cloudstate project yet? @viktorklang)

Reply:
I definitely don't know what's best. I found it interesting to comment, but I can't say if this is accepted by everyone in the project

Reply:
I think it is better to have a separate section on third-party tooling until we get to a state where something is official tooling?

Reply:
OK @viktorklang  I will remove mentions to the CLI